my-fund-backend/src/main/java/com/myfund/model/Budget.java

```diff
@@ -0,0 +1,21 @@
+package com.myfund.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@Entity
+@ToString
+@Table(name = "budget")
+public class Budget {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String nameBudget;
```

Comment:
`name` is fine, name`Budget` in Budget is redundant

Reply:
variable name changed